src/catalog/abstract_catalog.cpp

```diff
@@ -10,11 +10,12 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "binder/bind_node_visitor.h"
```

Comment:
Follow the style guide for the order of includes [here](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) (i.e., the first include should be `catalog/abstract_catalog.h` in `catalog/abstract_catalog.cpp`).